extensions/schema-compare/src/schemaCompareMainWindow.ts

```diff
@@ -299,33 +299,28 @@ export class SchemaCompareMainWindow {
 			columns: [
 				{
 					value: localize('schemaCompare.typeColumn', 'Type'),
-					toolTip: localize('schemaCompare.typeColumn', 'Type'),
```

Comment:
Is it a good idea to remove tool tip or 
1) should we change tooltip text to something more elaborative?
2) is there a way to set to not read tooltip explicitly?

Reply:
The column names seem pretty self explanatory, but we can add more description if it'll be helpful. @hannahqin thoughts on making more descriptive tooltips for the schema compare table columns?